app/Http/Controllers/InvitationLinkController.php

```diff
@@ -86,14 +90,42 @@ public function update(Request $request, InvitationLink $invitation_link): Respo
 
     }
 
+    public function saveIndex(Save $save): AnonymousResourceCollection {
+
+        // TODO Bitte die authorisierung prüfen
+        $this->authorize("view", $save);
+
+
+        return InvitationLinkResource::collection($save->invitationLinks);
+    }
+
+    public function acceptInvite(Request $request): Response {
+
+        $user = $request->user();
+        $invitationLink = InvitationLink::where('token', '=', '' . $request->token)->firstOrFail();
+        $save = Save::find($invitationLink->save_id);
+
+        $sharedSave = new SharedSave();
+        $sharedSave->user_id = $user->id;
+        $sharedSave->save_id = $save->id;
+
+        $sharedSave->permission = $invitationLink->permission;
+
+        $sharedSave->accepted = true;
+        $sharedSave->save();
```

Comment:
`$save->contributors()->save($sharedSave)`

Reply:
`$save->contributors()->save($sharedSave)` doesn't work.
It is possible to use `$save->contributors()->attach($user,["permission"=>$invitationLink->permission])`
accepted is handled automatically